tendrl/notifier/notification/handlers/snmp_handler.py

```diff
@@ -0,0 +1,251 @@
+from etcd import EtcdException
+from etcd import EtcdKeyNotFound
+from pysnmp.error import PySnmpError
+from pysnmp.hlapi import CommunityData
+from pysnmp.hlapi import ContextData
+from pysnmp.hlapi import ObjectIdentity
+from pysnmp.hlapi import ObjectType
+from pysnmp.hlapi import OctetString
+from pysnmp.hlapi import sendNotification
+from pysnmp.hlapi import SnmpEngine
+from pysnmp.hlapi import UdpTransportTarget
+from pysnmp.hlapi import usmDESPrivProtocol
+from pysnmp.hlapi import usmHMACMD5AuthProtocol
+from pysnmp.hlapi import UsmUserData
+from tendrl.commons.config import load_config
+from tendrl.commons.event import Event
+from tendrl.commons.message import ExceptionMessage
+from tendrl.commons.utils.log_utils import log
+from tendrl.notifier.notification import NotificationPlugin
+
+# USM users fixed snmp engine id
```

Comment:
USM?

Reply:
In SNMP USM means User-based Security Model